app/middlewares/passwordValidator.js

```diff
@@ -0,0 +1,12 @@
+const errors = require('../errors');
+
+const isAlphanumeric = string => string.match(/^[a-zA-Z0-9]+$/);
+
+const isValidPassword = password => password.length >= 8 && password.length <= 24 && isAlphanumeric(password);
+
+exports.handle = (req, res, next) => {
+  if (!isValidPassword(req.body.password)) {
+    next(errors.defaultError('Invalid password'));
```

Comment:
Remember, the error `defaultError` contains `internalCode:500`. If the user send a invalid password, you response with 500. This is very bad, you response must contain a internalCode 400